src/ert/_c_wrappers/enkf/enkf_fs.py

```diff
@@ -178,6 +181,52 @@ def copy_from_case(
             active,
         )
 
+    def save_summary_data(self, summary: "EclSum", realization: int):
+        import os
```

Comment:
For this PR (pushed to its own feature-branch) use pandas dataframe + HDF5. Performance testing afterwards.